src/main/java/org/moltimate/moltimatebackend/request/ActiveSiteAlignmentRequest.java

```diff
@@ -29,11 +30,15 @@
     private List<MultipartFile> customMotifs = new ArrayList<>();
     private String ecNumber; // TODO: Make this into a filter
 
-    public List<Structure> getPdbIdsAsStructures() {
-        return ProteinUtils.queryPdb(pdbIds);
+    public List<Structure> callPdbForStructures() {
+        List<Structure> structures = ProteinUtils.queryPdb(pdbIds);
+        if (structures.size() == 0) {
+            throw new InvalidPdbIdException(pdbIds);
```

Comment:
Won't this show all of the PDB Ids in the query, and not just the ones that did fail?

Reply:
If `ProteinUtils.queryPdb(pdbIds)` returns at least 1 structure then size > 0 and doesn't throw an error